web/src/features/AppConfig/components/AppConfig.tsx

```diff
@@ -491,49 +483,20 @@ class AppConfig extends Component<Props, State> {
   ): [ConfigGroup[], boolean] => {
     let hasValidationError = false;
 
-    // this is gated by showErrorsForEmptyValues which is set when save is attempted
-    const hasRequiredValidationError =
-      showErrorsForEmptyValues &&
-      Boolean(
-        groups?.find(
-          (group) =>
-            group?.when !== "false" &&
-            Boolean(
-              group?.items?.find(
-                (item) =>
-                  item.hidden !== true &&
-                  item.required &&
-                  !item.value &&
-                  item?.when !== "false"
-              )
-            )
-        )
-      );
-
     const newGroups = groups?.map((group: ConfigGroup) => {
       const newGroup = { ...group };
       const configGroupValidationErrors = validationErrors?.find(
         (validationError) => validationError.name === group.name
       );
 
-      if (configGroupValidationErrors || hasRequiredValidationError) {
+      // required errors are handled separately
+      if (group?.items?.find((item) => item.error)) {
+        newGroup.hasError = true;
+      }
```

Comment:
if there is a required item error in the group then we need to expand that group in the side nav so people can quickly get to the error